audio_service/pubspec.yaml

```diff
@@ -32,10 +32,7 @@ dependencies:
 
   audio_session: ^0.1.0
   rxdart: ^0.26.0
-  flutter_cache_manager: ^3.0.0-nullsafety.1
-  # This sqflite constraint is needed to make the >= 2.0.0 versions of
-  # flutter_cache_manager compile.
-  sqflite: ^2.0.0+2
```

Comment:
we should not mark transitive deps explicitly

Reply:
There was a temporary bug in flutter_cache_manager that required this, but it has since been fixed since `flutter_cache_manager: ^3.0.0` which has been applied to the `master` branch, although wasn't merged into `one-isolate` yet.

Reply:
should i revert this and you will apply the fix later?

Reply:
It's fine either way.

Reply:
i'll keep it, i just wasn't sure if fix for this problem has included more than just a `pubspec.yaml` change